components/Sections/Expertise.jsx

```diff
@@ -2,13 +2,14 @@ import expertiseList from "../../utils/expertiseData";
 export default function Expertise() {
   return (
     <section className="float-right">
-      <div className="-tracking-0.01">
-        <h4 className="text-2xl text-muted-300">Expertise</h4>
-        <div className="grid grid-cols-1 mt-5 gap-y-3">
+      <div className="text-xl leading-8 -tracking-0.01 md:text-2xl">
+        <h4 className=" text-muted-300">Expertise</h4>
+        <div className="grid grid-cols-1 mt-5 md:grid-cols-2 gap-y-3 gap-x-6">
           {expertiseList.map((expertise, index) => (
+            // used width of 70vw (w-70w) to intentionaly flow the text to 70% of the whole screen (wrapper margin included) but only on mobile below 400px
             <h5
               key={index}
-              className="text-xl leading-8 border-b border-muted-100 w-70w"
+              className="border-b border-muted-100 w-70w ssm:w-[18.125rem]"
```

Comment:
what do you mean by `ssm:` and also it would be better if you put value in the tailwind config

Reply:
It's a custom breakpoint added in `tailwind.config` 
I missed the arbitrary value though nice catch